apps/avatax/src/pages/api/cron/preheat-webhooks-handlers.ts

```diff
@@ -0,0 +1,83 @@
+import { BaseError } from "@/error";
+import { createLogger } from "@/logger";
+import { withOtel } from "@saleor/apps-otel";
+import { ResultAsync } from "neverthrow";
+import { NextApiRequest, NextApiResponse } from "next";
+
+const PreheatCheckoutCalculateTaxesError = BaseError.subclass("PreheatCheckoutCalculateTaxesError");
+const preheatOrderCalculateTaxesError = BaseError.subclass("PreheatOrderCalculateTaxesError");
+
+const logger = createLogger("preheatWebhookHandlersCron");
+
+const getBaseUrl = () =>
+  process.env.VERCEL_URL ? `https://${process.env.VERCEL_URL}` : "http://localhost:3000";
+
+const preheatCheckoutCalculateTaxes = () =>
+  ResultAsync.fromPromise(
+    fetch(`${getBaseUrl()}/api/webhooks/checkout-calculate-taxes`, {
+      method: "POST",
+      headers: {
+        "saleor-api-url": "https://mocked-cron.saleor.io/graphql/",
+        "saleor-event": "checkout_calculate_taxes",
+        "saleor-signature": "mocked-signature",
```

Comment:
suggestion: We could use `SALEOR_SIGNATURE_HEADER`, `SALEOR_EVENT_HEADER`, `SALEOR_DOMAIN_HEADER` / `SALEOR_API_URL_HEADER` consts from app-sdk

```ts
import { SALEOR_API_URL_HEADER } from "@saleor/app-sdk/const";
```